pczt/src/orchard.rs

```diff
@@ -67,6 +70,7 @@ pub struct Action {
     // These are required fields that are part of the final transaction, and are filled in
     // by the Constructor when adding an output.
     //
+    #[getset(get = "pub")]
     pub(crate) cv_net: [u8; 32],
     #[getset(get = "pub")]
     pub(crate) spend: Spend,
```

Comment:
Should `rcv` have `#[getset(get = "pub")]`?

Reply:
It was not necessary on the protocol-agnostic type for the sighash calculation (as it's private data, and useless for checking commitments without protocol awareness).